src/AccordionItem/index.js

```diff
@@ -149,7 +148,6 @@ export default class AccordionItem extends Component {
           [disabledClassName]: disabled
         }
       ),
-      role: 'tabpanel',
```

Comment:
I don't think the 'tablist/tabpanel' is appropriate for accordians; the ARIA reference version doesn't have this for example?

https://www.w3.org/TR/wai-aria-practices-1.1/#accordion